Atividades/para-casa-roberta/exercicio.js

```diff
@@ -0,0 +1,165 @@
+class AnimalDomestico {
+  constructor(nome, idade, cor, castrado) {
+    this.nome = nome;
+    this.idade = idade;
+    this.cor = cor;
+    this.consultas = [];
+    this.vacinas = [];
+    this.castrado = castrado;
+  }
+}
+
+class Gato extends AnimalDomestico {
+  constructor(nome, idade, cor, castrado, externo, social) {
+    super(nome, idade, cor, castrado);
+    this.externo = externo;
+    this.social = social;
+    this.consultas = [];
+    this.vacinas = [];
+  }
+  miar() {
+    console.log("miau miau");
+  }
+  brincar() {
+    console.log(`${this.nome} use uma varinha`);
+  }
+  acariciar() {
+    if (this.social == true) {
+      console.log("ron ron ron");
+    } else {
+      console.log("schiiii");
+    }
+  }
+  vacinar(vacina) {
+    this.vacinas.push(vacina);
+    this.consultas.unshift(new Date());
+  }
```

Comment:
Acho que o seu não tá incluindo a vacina no campo de consulta, porque tá faltando um argumento em vacinar. Nesse caso, teria que colocar vacina e data como argumentos. 

No início o meu também não tava funcionando, até que percebi que poderia ser isso e deu certo!!